test/dummy/db/migrate/20201112193021_create_dynamo_db_sessions_table.rb

```diff
@@ -0,0 +1,9 @@
+class CreateDynamoDbSessionsTable < ActiveRecord::Migration[6.0]
```

Comment:
Ditto on this - I think we'd end up with a new file each time from the test w/ differnt date stamps?

Reply:
Rails is smart enough to check the contents (and sometimes the name) so even running at a later date will determine that the file is identical and not create a new one.